src/main/java/org/mamute/auth/LdapSSOFilter.java

```diff
@@ -0,0 +1,39 @@
+package org.mamute.auth;
+
+import javax.enterprise.event.Observes;
+import javax.inject.Inject;
+import javax.servlet.http.HttpServletRequest;
+
+import org.mamute.dao.UserDAO;
+import org.mamute.model.LoggedUser;
+import org.mamute.model.User;
+
+import br.com.caelum.vraptor.environment.Environment;
+import br.com.caelum.vraptor.events.MethodReady;
+
+public class LdapSSOFilter {
+
+	@Inject private Environment env;
+	@Inject private UserDAO users;
+	@Inject private LDAPApi ldap;
+	@Inject private LoggedUser loggedUser;
+	@Inject private HttpServletRequest request;
+	@Inject private Access system;
+
+	public void checkSSO(@Observes MethodReady methodReady) {
+		if (env.supports(LDAPApi.LDAP_SSO) && !loggedUser.isLoggedIn() && request.getRequestURI() != null
+				&& !request.getRequestURI().endsWith("/logout")) {
+			String userName = request.getRemoteUser();
```

Comment:
Right now, the only way to use this feature is by enabling this filter? We usually configure things through properties files, but if that is too hard, then let it be. Just make sure to document how to configure this SSO feature. Maybe you can add a question to meta.mamute.org and add an answer documenting it, and reference the question in this PR.

What do you think @falbrech-hsdg?